uploader.json

```diff
@@ -388,6 +388,11 @@
             "pattern": [],
             "symbolic_name": "ls_var_spool_clientmq"
         },
+        {
+             "command": "/bin/ls -l /var/lib/cni/networks/openshift-sdn",
+             "pattern": [],
+             "symbolic_name": "ls_var_lib_cni_networks_openshift-sdn"
```

Comment:
@shzhou12 this symbolic name needs to match the name of the spec in insights core `ls_ocp_cni_openshift_sdn`.